src/components/Rubric.js

```diff
@@ -174,4 +174,4 @@ const mapDispatchToProps = (dispatch) => {
   return bindActionCreators(Actions, dispatch);
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(Rubric);
+export default withRouter(connect(mapStateToProps, mapDispatchToProps)(Rubric));
```

Comment:
Nice. I notice that `withRouter` from `react-router` to inject the history and other props from react route